onnx/test/large_onnx_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+import unittest
+
+import numpy as np
+
+from onnx import ModelProto, TensorProto, checker, load_model
+from onnx.external_data_helper import _get_all_tensors, uses_external_data
+from onnx.helper import make_graph, make_model, make_node, make_tensor_value_info
+from onnx.large_helper import (
+    LargeModelContainer,
+    LargeModelFileFormat,
+    make_large_model,
+    make_large_tensor_proto,
+)
+from onnx.numpy_helper import from_array
+
+
+class TestLargeOnnx(unittest.TestCase):
+    @staticmethod
```

Comment:
Avoid staticmethods - I would convert to a module level private function

Reply:
done